Misc/NEWS.d/next/Library/2020-11-17-14-30-12.bpo-42383.ubl0Y_.rst

```diff
@@ -0,0 +1,2 @@
+Fix pdb: previously pdb would fail to restart the debugging target if it was
```

Comment:
a PR own a misc file is enough. in here

Reply:
> a PR own a misc file is enough. in here

I do not understand. Are you saying I should merge the two misc files into one?

Reply:
yes, you are right.

Reply:
IMHO, one PR always means a bugfix or a  sepeareted developing feature, so a single misc file is good enough.

Reply:
Here is the situation: I initially treated pdb as a black box. It had two separate bugs. One bug affected how sys.path[0] was populated while the other prevented pdb from properly restarting the target. So I opened two bugs. Then I looked at the code and it just so happens that changing one line fixes both bugs. I'm not sure what to do now.

Reply:
My personal experience(NOT the official way):
*  Creating the first PR to solve the fist issue you want fixed.
*  After the first PR merged, we can add some description info in second issue, something like: `After PR-XXX merged, this issue should be resolved by the way.`. MAYBE you can create an another PR to add the testcase of the second issue.

Reply:
Following this instructions

https://github.com/python/cpython/pull/23338#discussion_r526146998

> My personal experience(NOT the official way):
> 
>     * Creating the first PR to solve the fist issue you want fixed.
> 
>     * After the first PR merged, we can add some description info in second issue, something like: `After PR-XXX merged, this issue should be resolved by the way.`. MAYBE you can create an another PR to add the testcase of the second issue.

I have updated the pull request.